catalog/spatial/registry/pom.xml

```diff
@@ -158,7 +158,7 @@
             <dependency>
                 <groupId>com.thoughtworks.xstream</groupId>
                 <artifactId>xstream</artifactId>
-                <version>1.4.4</version>
+                <version>1.4.9</version>
```

Comment:
You could use dependency management in the spatial/pom.xml so we wouldn't need to upgrade the version in the registry pom anymore. But maybe we want to support different versions.